pintereach/src/App.js

```diff
@@ -1,29 +1,75 @@
-import React from 'react';
+import React, { useReducer, useState } from "react";
 import { BrowserRouter as Router, Route, Link, Switch } from "react-router-dom";
+import PrivateRoute from "./components/PrivateRoute";
+import { Reducer, appState } from "./reducers/Reducer";
+import { ArticlesContext } from "./contexts/ArticlesContext";
 
 import Login from "./components/Login";
 import Signup from "./components/Signup";
 import Articles from "./components/Articles";
-import PrivateRoute  from "./components/PrivateRoute";
-
-import './App.css';
+import BoardForm from "./components/BoardForm";
+import UserHome from './components/UserHome.js'
 
+import "./App.css";
 
 function App() {
-  
-  return (
+	const [state, dispatch] = useReducer(Reducer, appState);
+
+	//Category state
+	const [category, setCategory] = useState([
```

Comment:
rather than setting a state here for the categories, you can just pull it in from the reducer file